### PR TITLE
Documentation changes: Quickstart

### PR DESCRIPTION
**Pull request by macieksmuga**
The old `docs/README.md` file has been moved to `ArchitectureOverview.md` and the new `README` is now a quickstart aimed at GA4GH developers interested in creating tests for the suite, as opposed to working on the architecture of the compliance suite itself.

### CODE REVIEW

**Comment by diekhans** [comment]
I would really like to have a traditional structure where a README is just a REAMDME, a simple explanation of what the software is and why we should care and what they should read if they do care

The a docs directory with things like QuickStart.md, IDontReallyCare.md, etc.

**Comment by macieksmuga** [comment]
There's already a `README` at the top level of the project. I can of course rename this to `QuickStart.md` - I think people would figure out it's a decent place to look if they need a quick... um... start.

**Comment by diekhans** [comment]
Oh, OK< this is in docs.  Yes QuickStart.md is much more obvious
what it is.

File: docs/Quickstart.md [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+Before running any tests, you may want to set an environment variable to record the server URL you’re testing against:
+
+     > export CTK_TGT_URLROOT=“<your GA4GH-compatible server URL>"
+
+You may also wish to set the dataset ID for the test dataset as an environment variable. Note that the ID given below is specific to the GA4GH reference server running on the compliance dataset, and will likely be different on another
+server implementation.
+
+     > export CTK_TGT_DATASET_ID=“YnJjYTE="
+
+It is also possible to set these parameters on a per-run basis
```
Comment (hjellinek) [inline docs/Quickstart.md]: Period.

File: docs/Quickstart.md [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+For example, the fourth run of the tests on
+my local server would display its results here:
+
+     <test_directory>/testresults/localhost_8000/00004/report/html/index.html
+     
+_For more information, see the [command line intro](README-IntroAtCommandLine.txt) detailed documentation._
+
+## write, compile & run a test
+
+The java source code files containing the actual tests are all located in:
```
Comment (hjellinek) [inline docs/Quickstart.md]: Java with a capital J.  Not to be too pedantic. :-)

File: docs/RunningTests_CLI.md [hunk 3]
```diff
@@ -1,135 +1,102 @@
...
 
 and see that you can run with just:
+
     `./ctk`
 
-This will use defaults and environment variables and property files. If you want to override some properties, add the property and new value to the command line:
+This will use defaults and environment variables and property files. If you want to override some properties, add the property and new value to the command line.
+For example:
 
     ./ctk --ctk.tgt.urRoot=http://myserver:8000
```
Comment (hjellinek) [inline docs/RunningTests_CLI.md]: Typo: ctk.tgt.urlRoot

**Pull request merged**